packages/modules/src/modules/banners/momentTemplate/components/MomentTemplateBannerBody.tsx

```diff
@@ -31,13 +28,15 @@ export function MomentTemplateBannerBody({
         <div css={styles.container}>
             <Hide above="tablet">
                 {createBannerBodyCopy(
-                    mobileParagraphs ?? paragraphs,
-                    mobileHighlightedText ?? highlightedText,
+                    mobileContent.paragraphs,
+                    mobileContent.highlightedText,
```

Comment:
Sometimes we were handling the fallback in the parent component, and sometimes within component itself. By having it right at the top in the wrapper, individual components can now just render the mobile content at mobile and are unconcerned with if that's actually falling back on the desktop content